supabase/migrations/20240906043335_release_attachments.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE public.release_attachments (
+    release_id UUID NOT NULL REFERENCES public.releases ON DELETE CASCADE,
+    attachment_id UUID NOT NULL,
+    name TEXT NOT NULL,
+    file_path TEXT NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    PRIMARY KEY (release_id, attachment_id),
+    CONSTRAINT unique_file UNIQUE (file_path)
+)
```

Comment:
If needed in the future, the database model can be migrated to include a `file` entity. 

The `release_attachments` table would then have `release_id` and `file_id` fields (along with any other attachment-specific fields).